docs/source/models/engine_args.rst

```diff
@@ -0,0 +1,68 @@
+.. _engine_args:
+
+Engine Arguments
+==================
+
+Below, you can find an explanation of every engine argument for vLLM:
+
+* --model
+    Name or path of the huggingface model to use.
+* --tokenizer
+    Name or path of the huggingface tokenizer to use.
+* --revision
+    The specific model version to use. It can be a branch name, a tag name, or a commit id. If unspecified, will use the default version.
+* --tokenizer-revision
+    The specific tokenizer version to use. It can be a branch name, a tag name, or a commit id. If unspecified, will use the default version.
+* --tokenizer-mode
+    {auto,slow} The tokenizer mode.
+    
+    * -- "auto" will use the fast tokenizer if available
+    * -- "slow" will always use the slow tokenizer.
+* --trust-remote-code
+    Trust remote code from huggingface.
+* --download-dir
+    Directory to download and load the weights, default to the default cache dir of huggingface
+* --load-format
+    {auto,pt,safetensors,npcache,dummy} The format of the model weights to load. 
+
+    * -- "auto" will try to load the weights in the safetensors format and fall back to the pytorch bin format if safetensors format is not available. 
+
+    * -- "pt" will load the weights in the pytorch bin format. "safetensors" will load the weights in the safetensors format. 
+
+    * -- "npcache" will load the weights in pytorch format and store a numpy cache to speed up the loading. 
+
+    * -- "dummy" will initialize the weights with random values, which is mainly for profiling.
+* --dtype
+    {auto,half,float16,bfloat16,float,float32} Data type for model weights and activations. 
+
+    * -- The "auto" option will use FP16 precision for FP32 and FP16 models, and BF16 precision for BF16 models.
+    * -- Use "half" for AWQ quantization.
+
+* --max-model-len
+    Model context length. If unspecified, will be automatically derived from the model config.
+* --worker-use-ray
+    Use Ray for distributed serving, will be automatically set when using more than 1 GPU.
+* --pipeline-parallel-size
+    (-pp) Number of pipeline stages.
```

Comment:
Can you make it more clear that `-pp` is an alias? Similar to `-tp` below.

Reply:
Added that they are an alias